src/components/missions/MissionMember.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+const MissionMember = ({ joinId }) => {
+  const getStatus = useSelector((state) => state.missionStore);
+
+  let res;
+  getStatus.forEach((v) => {
+    if (v.reserve === true && v.id === joinId) {
+      res = v.reserve;
+    }
+  });
+
+  return (
+    <div>
+      {res === true ? (
+        <p className="success">Active Member</p>
+      ) : (
+        <p className="danger">Not a Member</p>
+      )}
+    </div>
+  );
```

Comment:
- [ ] The same thing here. Kindly use React conditional rendering syntax to show the Active Member or Not a Member. The syntax looks like this 👉 
```javascript
{rocket.reserved && ( 
    // render Cancel Rocket button
)}
```